index.html

```diff
@@ -20,16 +20,10 @@
   <link rel="import" href="../polymer/polymer.html">
   <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
 
-  <style>
-    body {
-      margin: 16px;
-    }
-  </style>
-
 </head>
 <body>
 
-  <iron-doc-viewer src="paper-radio-button.html"></iron-doc-viewer>
```

Comment:
`<src>` is sooooo yesterday. `<iron-doc-viewer>` picks it up automagically now. :)

(see https://github.com/PolymerElements/iron-doc-viewer/blob/master/iron-doc-viewer.html#L39)